src/db/db_holiday.erl

```diff
@@ -29,3 +31,35 @@ create(Country, Date, Name) ->
     {ok, [Result | []]} -> {ok, Result};
     {error, unique_violation} -> {error, holiday_already_exists}
   end.
+
+get_user_holidays(Email) ->
+    Q = <<"SELECT to_char(date, 'YYYY-MM-DD') as date, name FROM user_holidays "
+          "WHERE \"user\" = (SELECT id FROM users WHERE email = $1) "
+          "ORDER BY date">>,
+    QDefault = <<"SELECT to_char(date, 'YYYY-MM-DD') as date, name FROM holidays "
```

Comment:
this doesn't work, copy the default country holidays to the user_holidays table when the user is created